bin/reth/src/node/events.rs

```diff
@@ -127,6 +128,14 @@ impl NodeState {
 
                 info!(number=block.number, hash=?block.hash, "Block added to canonical chain");
             }
+            BeaconConsensusEngineEvent::ChainCanonicalized(outcome, elapsed) => match outcome {
+                CanonicalOutcome::AlreadyCanonical { header } => {
+                    info!(number=header.number, header=?header.hash, ?elapsed, "Chain is already canonical");
```

Comment:
should this be a warning?

Reply:
this commonly happens on start up and could also happen if there are more CL connected

info is fine here imo

Reply:
i'd prefer downgrading it to trace/debug level, but no strong opinion on this

Reply:
agree on debug level, because it might be seem like something's wrong while in reality it was a no-op